week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -11,9 +20,13 @@ function mood() {
   }
 }
 
+// 2) For any numerical value greater or equal to 10
+//    Hint: use constant `num` and only change isBigEnough.
+//    Variable isBigEnough needs to evaluate to a boolean
+
 function greaterThan10() {
-  let num = 10;
-  let isBigEnough;
+  const num = 10;
+  const isBigEnough = num >=num ;
```

Comment:
Here Sulaiman this variable should check if num is greater than a number that you choose and use this variable in the if/else statement to get the appropriate answer. It needs to be fixed.

Reply:
got it